src/components/UserProfile/UserProfile.jsx

```diff
@@ -1,4 +1,3 @@
-import React, { useState, useEffect, useRef } from 'react';
```

Comment:
Still missing this line

Reply:
Just add it back should be fine. Alan deleted it accidentally in his most recent commit.

Reply:
Thanks for bringing this to my attention, @KaixiangGU. I've added the missing line back to the code in my branch, so it should be included in the latest version of the PR.

Reply:
Thank you! Looks good!